src/utils/webSockets.service.js

```diff
@@ -4,78 +4,80 @@ import { Observable } from 'rxjs';
 import { map, filter } from 'rxjs/operators';
```

Comment:
there doesn't seem to be any relevant change on this file, please remove this change.